server/services/caseloadService.ts

```diff
@@ -203,4 +217,14 @@ export default class CaseloadService {
     }
     return LicenceType.AP_PSS
   }
+
+  private filterActiveLicencesIfVariationExists = (licences: LicenceSummary[]): LicenceSummary[] => {
+    return licences.filter(licence => {
```

Comment:
Not sure how this is checking whether a variation exists? It's filtering ACTIVE ok, but how does it know whether there is a variation present?

Reply:
The only other statuses that can go in here are         

```
LicenceStatus.VARIATION_IN_PROGRESS,
LicenceStatus.VARIATION_SUBMITTED,
LicenceStatus.VARIATION_APPROVED,
```

Reply:
So this filter allows a licence through if it is not in ACTIVE state. If it is in active state, then check if there are any other licences in the list which belong to that person. If there are then it will get filtered out